scripts/CSSRE/under-min-isr-partitions/metadata.yaml

```diff
@@ -33,6 +33,12 @@ rbac:
         - "kafka.strimzi.io"
       resources:
         - "kafkas"
+    - verbs:
+        - "list"
+      apiGroups:
+        - "core.strimzi.io"
+      resources:
+        - "strimzipodsets"
```

Comment:
Given the proposed change to scripts/CSSRE/lib/py/kafka.py switches from using statefulsets to service, why do are RBAC permissions for`strimzipodsets` required?

Reply:
Hello Keith, thank you! That does make more sense. 
I've updated the metadata files for the script in this pr to have permissions for services and removed statefulset permissions. I also changed the permissions on the [changes I made recently to the gather-kafka-instance-state script](https://github.com/openshift/managed-scripts/pull/82), because I had done the same thing there. I've tested it locally and it seems to still work as expected.